runtime/src/lib.rs

```diff
@@ -265,6 +265,7 @@ impl pallet_evm::Config for Runtime {
 	type Runner = pallet_evm::runner::stack::Runner<Self>;
 	type Precompiles = precompiles::MoonbeamPrecompiles<Self>;
 	type ChainId = EthereumChainId;
+	type OnChargeTransaction = ();
```

Comment:
This would mean we weren't charging transaction fees (except for evm transactions). Given the importance of transaction fees. We should have a test for this. For example: Make a system.remark transaction and assert that the balance has decreased.

Reply:
MOON-410

Reply:
Oh, I'm so wrong. This is pallet EVM. I thought this was pallet_transaction_payment.